playbooks/appserver.yml

```diff
@@ -9,10 +9,6 @@
       when: consul_servers | length > 0
     - role: node-exporter
       when: NODE_EXPORTER_PASSWORD is not none
-    - role: filebeat
-      # Appservers may have some Elastic stuff already installed on them.
-      filebeat_elastic_apt_pin: 200
-      when: filebeat_logstash_hosts | length > 0
```

Comment:
I don't think we use the `appserver.yml` book anymore (it was only used by Ocim), so we could remove the entire playbook.

Reply:
No, we don't, but I'd like to keep the PR "focused" as this repo will be archived anyways.